### PR TITLE
{bio}[foss/2019b] RevBayes + deps

### PR DESCRIPTION
**Pull request by YensyGomez**

### CODE REVIEW

**Review by Micket — CHANGES_REQUESTED** [review]
Does this require Boost 1.74? If not, please use 1.71.0 that already exists for 2019b, since it would make this module more compatible with other 2019b modules.
If it does require it, then there is already a newer bzip2 version for GCCcore 8.3.0 so a foss version should be added.

File: easybuild/easyconfigs/r/RevBayes/RevBayes-1.1.1-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/revbayes/revbayes/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['d61293fceac817d8203ed1e828661d76c73fa16bf04458a50a37057e99fd40c0']
+
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('Boost', '1.71.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/r/RevBayes/RevBayes-1.1.1-foss-2019b.eb:28]: @YensyGomez It doesn't make sense to include `Boost` as a dependency twice, only via `dependencies` is fine (unless `Boost` is only a build dependency, then only via `builddependencies` of course)

**Review by boegel — CHANGES_REQUESTED** [review]
@YensyGomez Would it be possible to include a simple test command like `rb --help` in `sanity_check_commands`?

**Comment by Micket** [comment]
We have 2 RevBayes PRs right now, and they differ slightly.
#12411 includes Java, and has slightl configure flags. I don't know which is more correct

**Comment by boegel** [comment]
Closing this PR since there's no reply at all after the review...

**Pull request closed without merging**